packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelForm.tsx

```diff
@@ -76,6 +78,29 @@ function renderDefaultValue(
   return defaultValue
 }
 
+const getRecommendedMaxSize = (
+  defaultAnnotationModel: ColumnModel | null | undefined,
+): number | null | undefined => {
+  return defaultAnnotationModel ? defaultAnnotationModel.maximumSize : null
+}
+
+const getCurrentMaxSize = (columnModel: ColumnModelFormData): number | null => {
+  return columnModel.maximumSize !== undefined
+    ? parseInt(columnModel.maximumSize.toString(), 10)
+    : null
```

Comment:
Small changes for null safety, in case `maximumSize` is `null` instead of `undefined`
```suggestion
  return columnModel.maximumSize != null
    ? parseInt(String(columnModel.maximumSize), 10)
    : null
```